device/esp_tinyusb/idf_component.yml

```
@@ -9,7 +9,7 @@ targets:
   - esp32p4
   - esp32h4
 dependencies:
-  idf: '>=5.0' # IDF 4.x contains TinyUSB as submodule
+  idf: '>=5.4' # IDF 4.x contains TinyUSB as submodule
```

The component still can be used with the release v5.0 and for all others esp-idf releases, that are in maintenance period. So, it is not necessary to cut off the earlier releases. 

```suggestion
  idf: '>=5.0' # IDF 4.x contains TinyUSB as submodule
```